pixi.toml

```diff
@@ -3,10 +3,10 @@
 # -*- mode: toml -*-
 
 [project]
-name = "tiledbsoma-feedstock"
+name = "cf-tiledbsoma-feedstock"
```

Comment:
That's annoying. When I rerendered locally, it was contaminated by the local name of my Git repository (I have to use a different name since my clone of TileDB-Inc/tiledbsoma-feedstock has a completely different Git history).